specification/validated-relationships-service-api.yaml

```diff
@@ -199,6 +230,236 @@ paths:
                 $ref: '#/components/schemas/OperationOutcome'
 components:
   schemas:
+    QuestionnaireResponse:
+      type: object
+      description: A FHIR QuestionnaireResponse
+      properties:
+        resourceType:
+          type: string
+          description: The FHIR resource type
+          enum: ["QuestionnaireResponse"]
+        status:
+          type: string
+          description: The position of the questionnaire response within its overall lifecycle.
+          enum: ["completed"]
+        authored: 
+          type: string
+          description: The date and/or time that this set of answers were last changed.
+          # On Confluence the example has this as a string but here:
```

Comment:
Note: all comments can't be seen on Swagger editor so I presume also developer portal. Close this comment that's the expected behaviour

Reply:
Yes just for internal consumption